app/View/Components/TranslatableInput.php

```diff
@@ -56,12 +56,26 @@ class TranslatableInput extends Component
      */
     public $required;
 
+    /**
+     * The name for asl or lsq content.
+     *
+     * @var bool
+     */
+    public $interpretationName;
+
+    /**
+     * The namespace for asl or lsq content.
+     *
+     * @var bool
+     */
+    public $interpretationNameSpace;
+
```

Comment:
Should make these nullable and they are strings not boolean values. `@var string` and `public ?string $interpretationName` and `public ?string $interpretationNameSpace`.